stream.go

```diff
@@ -1588,6 +1589,8 @@ type serverStream struct {
 
 	sendCompressorName string
 
+	recvFirstMsg bool // recv frist msg from client
```

Comment:
This comment doesn't really say anything.  How about:

```suggestion
	recvFirstMsg bool // set after the first message is received
```

Reply:
Done.

Reply:
Sorry, I missed committing the changes.